.dumi/theme/builtins/IconSearch/fields.ts

```diff
@@ -13,26 +13,35 @@ const chain = [
   'Polygon',
   'Sui',
   'Tron',
-  'USDT',
   'Base',
   'Avalanche',
+  'Fantom',
+  'Celo',
+  'Cronos',
+  'Moonbeam',
 ];
 
+const token = ['DAI', 'WBTC', 'AAVE', 'USDT'];
+
 const tool = [
   'CoinbaseWallet',
   'Etherscan',
   'MetaMask',
   'WalletConnect',
   'TokenPocket',
   'Safeheron',
+  'Uniswap',
+  'PancakeSwap',
+  'SushiSwap',
```

Comment:
<img width="1113" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/65b9367a-1ad2-4870-97d4-19a6a07e3144">

Due to duplicate IDs, when multiple icons are present on the page, there may be display issues.

You can fix it like: https://github.com/ant-design/ant-design-web3/pull/346

Reply:
Thank you for the suggestion. The issue has been fixed.
<img width="934" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/37927942/bf2f369f-4dd8-49ab-93f2-c61a23f1d0bc">